src/features/bookings/components/BookingDetailsTicketContent.web.test.tsx

```diff
@@ -28,60 +30,52 @@ describe('BookingDetailsTicketContent', () => {
   })
 
   describe('proDisableEventsQrcode & QR code display', () => {
-    it('should display the QR code when proDisableEventsQrcode is false', () => {
-      const booking = bookingsSnap.ongoing_bookings[0]
-      booking.stock.offer.isDigital = false
-      const { getByTestId } = render(
-        <BookingDetailsTicketContent booking={booking} proDisableEventsQrcode={false} />
-      )
-      getByTestId('qr-code')
-    })
-
-    it('should not display the QR code when proDisableEventsQrcode is true', () => {
-      const booking = bookingsSnap.ongoing_bookings[0]
-      booking.stock.offer.isDigital = false
-      const { queryByTestId } = render(
-        <BookingDetailsTicketContent booking={booking} proDisableEventsQrcode={true} />
-      )
+    const initialBooking = bookingsSnap.ongoing_bookings[0]
+    initialBooking.stock.offer.isDigital = false
 
-      expect(queryByTestId('qr-code')).toBeFalsy()
-    })
+    it('should not display button for email consulation if by email collect and delay is specified and email normally received', () => {
+      const booking = {
+        ...initialBooking,
+        confirmationDate: formatISO(new Date()).slice(0, -1),
+        stock: {
+          ...initialBooking.stock,
+          beginningDatetime: formatISO(addDays(new Date(), 2)).slice(0, -1),
+          offer: {
+            ...initialBooking.stock.offer,
+            isDigital: false,
+            withdrawalType: OFFER_WITHDRAWAL_TYPE_OPTIONS.BY_EMAIL,
+            withdrawalDelay: 60 * 60 * 24 * 2,
+          },
+        },
+      }
 
-    it('should not display on site collect wording if type collect is not on site', () => {
-      const booking = bookingsSnap.ongoing_bookings[0]
-      booking.stock.offer.isDigital = false
       const { queryByTestId } = render(
         <BookingDetailsTicketContent booking={booking} proDisableEventsQrcode={true} />
       )
 
-      expect(queryByTestId('collect-info')).toBeFalsy()
+      expect(queryByTestId('collect-info-email-btn')).toBeFalsy()
     })
 
-    it('should display on site collect wording', () => {
-      const booking = bookingsSnap.ongoing_bookings[1]
-      booking.stock.offer.isDigital = false
-      const { getByTestId } = render(
-        <BookingDetailsTicketContent booking={booking} proDisableEventsQrcode={true} />
-      )
-      getByTestId('collect-info')
-    })
+    it('should not display button for email consulation if by email collect and the event is today', () => {
```

Comment:
il me semble qu'il y a des fautes (à plusieurs endroits)

```suggestion
    it('should not display button for email consultation if by email collect and the event is today', () => {
```